src/components/DatePicker/index.android.js

```diff
@@ -45,13 +47,18 @@ class DatePicker extends React.Component {
                 <TextInput
                     label={this.props.label}
                     value={dateAsText}
```

Comment:
Can we remove `value` altogether? Similar to what we did to `TextInput` here https://github.com/Expensify/App/commit/c66234c7264ec13e53f6a5e0f5e2ca335180e593?